src/Servers/Kestrel/Core/test/HttpRequestHeadersTests.cs

```diff
@@ -3,11 +3,13 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
 using Microsoft.Extensions.Primitives;
 using Xunit;
+using static CodeGenerator.KnownHeaders;
```

Comment:
Is this being used? Does KnownHeaders still need to be in the shared folder?

Reply:
Yes; is used in `[MemberData(nameof(KnownRequestHeaders))]` to check all the `KnownRequestHeaders` work as expected.